tests/mechanic/provisioner_test.py

```diff
@@ -690,22 +692,28 @@ def test_provisioning_with_defaults(self, uuid4):
         soft: -1
         hard: -1
     volumes:
-      - %s:/usr/share/elasticsearch/data
-      - %s:/var/log/elasticsearch
-      - %s:/usr/share/elasticsearch/heapdump
+      - {data_dir}:/usr/share/elasticsearch/data
+      - {log_dir}:/var/log/elasticsearch
+      - {heap_dump_dir}:/usr/share/elasticsearch/heapdump
     healthcheck:
       test: nc -z 127.0.0.1 39200
       interval: 5s
       timeout: 2s
-      retries: 10"""
-            % (data_dir, log_dir, heap_dump_dir),
+      retries: 10
+    networks:
+      - rally-es
+networks:
+  rally-es:
+    driver_opts:
+      com.docker.network.bridge.host_binding_ipv4: "{node_ip}\"""",
```

Comment:
not: to avoid this backslashing we can also change the f-string from `"""` to `'''`.

Reply:
Good point. Addressed in https://github.com/elastic/rally/pull/1313/commits/46786ddca5d67efd9ee8df3c7b05fe2872e2281b.